app/uk/gov/hmrc/digitalservicestax/frontend/Country.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.digitalservicestax.data
+
+import play.api.libs.json._
+
+
+object Country {
+
+  sealed abstract case class Country private(name: String, code: String)
+  object Country {
+    def unQuote(quotedString: String): String =
+      quotedString.slice(1,quotedString.length-1)
+
+    def apply(name: String, code: String): Country = {
+      new Country(unQuote(name), unQuote(code).replace("country:","")){}
+    }
+  }
+
+  val countriesJson: JsValue = {
```

Comment:
Are you planning to only have the Country object on the frontend? 
I'd be inclined to have a domain model that is shared between the FE and BE.

Reply:
actually thinking about it, ignore my previous remark - it's not really a domain model so much as an input parser....